ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/errors/KVErrors.scala

```diff
@@ -62,13 +62,18 @@ object KVErrors extends LedgerApiErrorGroup {
           ErrorCategory.InvalidGivenCurrentSystemStateOther, // It may succeed at a later time
         ) {
       case class Reject(
-          minimum_record_time: Instant,
-          maximum_record_time: Instant,
+          minimumRecordTime: Instant,
+          maximumRecordTime: Instant,
       )(implicit loggingContext: ContextualizedErrorLogger)
           extends KVLoggingTransactionErrorImpl(
             cause =
-              s"Invalid ledger time: Record time is outside of valid range [$minimum_record_time, $maximum_record_time]"
-          )
+              s"Invalid ledger time: Record time is outside of valid range [$minimumRecordTime, $maximumRecordTime]"
+          ) {
+        override def context: Map[String, String] = Map(
```

Comment:
It really concerns me that we could easily forget to override `context` for a particular error. I honestly think that having the logic in `BaseError` is dangerous.

To mitigate this, I suggest adding `override def context = kvContext` in `KvLoggingTransactionErrorImpl` and making `kvContext` an abstract method that _must_ be overridden by subclasses.

Reply:
I share your concern, especially that the logic in `BaseError` is not transparent.
I have made changes similar to as you suggested, with the slight change that `KvLoggingTransactionErrorImpl` now explicitly overrides `context` with an empty `Map` so subclasses must override it to produce any error metadata.
Does this work for you?

Reply:
Yes, sounds good!